.github/workflows/_build.yaml

```diff
@@ -31,7 +31,7 @@ on:
         required: true
       DOCKERFILE:
         type: string
-        description: "Dockerfile to use, e.g. .github/container/Dockerfile.t5x.amd64"
```

Comment:
Why renaming all the amd64 Dockerfiles? Is there a discussion that I missed? :grin:

Reply:
The difference between amd64 and arm64 is that we build some wheels (like lingo, tf-text) for arm64, but not for amd64. With this PR we build these packages for both archs, so no need to have 2 different Dockerfiles. I remove arm64 version of Dockerfile and extension `amd64`.

Reply:
I would view manual building of lingvo/tftext wheel as a temporary remedy that (unfortunately) complicates the Dockerfile. This workaround shouldn't stay forever. We should push for proper wheel builds by upstream and at the meantime keep the simpler Dockerfiles so that they will become the default again hopefully soon.

Reply:
Agree